docker/dockerfiles/be/be_entrypoint.sh

```diff
@@ -127,7 +127,12 @@ if [[ "x$LOG_CONSOLE" == "x1" ]] ; then
 fi
 
 while true; do
-  $STARROCKS_HOME/bin/start_be.sh $addition_args
+  $STARROCKS_HOME/bin/start_be.sh $addition_args &
+  be_process_pid=$!
+  # forward SIGTERM to be process_pid explicitly 
+  trap "kill -TERM $be_process_pid 2>/dev/null" TERM
```

Comment:
possibly using stop_be.sh as the SIGTERM trap, no need to get the process pid and do the manual killing.